base_environment.yml

```diff
@@ -2,109 +2,108 @@ name: base
 channels:
   - conda-forge
 dependencies:
-  - beautifulsoup4=4.8.1=py37_0
-  - boto3=1.10.34=py_0
-  - bqplot=0.12.1=py_0
-  - bumpversion=0.5.3=py_1001
-  - cachecontrol=0.12.5=py_0
-  - cartopy=0.17.0=py37h423102d_1009
-  - cftime=1.0.4.2=py37hc1659b7_0
-  - click=7.0=py_0
-  - compilers=1.0.4=0
-  - dask=2.8.1=py_0
-  - dask-glm=0.2.0=py_1
-  - dask-ml=1.1.1=py_0
-  - datashader=0.8.0=py_0
-  - distributed=2.8.1=py_0
-  - dropbox=9.4.0=py_0
+  - beautifulsoup4
+  - bottleneck
+  - boto3
+  - bqplot
+  - bumpversion
+  - cachecontrol
+  - cartopy
+  - cftime
+  - click
+  - compilers
+  - dask
+  - dask-glm
+  - dask-kubernetes
+  - dask-ml
+  - datashader
+  - descartes
+  - distributed
+  - dropbox
 # need to make sure we get esmpy compiled with mpi otherwise xesmf regridding
 # won't work
-  - esmpy=8.0.0=mpi_mpich_py37ha9b28fa_101 # pinkeep: esmpy=8.0.0=mpi_mpich_py37ha9b28fa_101
-  - fastparquet=0.3.2=py37hc1659b7_0
-  - fiona=1.8.13=py37h900e953_0
-  - fusepy=3.0.1=py_0
+  - esmpy
```

Comment:
if we end up still needing to pin this i think we can do something like esmpy=*=mpi*

Reply:
would we rather have `esmpy==mpi` than the minor version? (`esmpy==7.1`)

Reply:
nope I think we can do the minor version now! see [this page](https://xesmf.readthedocs.io/en/latest/installation.html?highlight=mpi#install-on-local-machine-with-conda)